checker/tests/resourceleak/MustCallAliasNormalExit.java

```diff
@@ -21,7 +21,7 @@ public void close() throws IOException {
     this.is.close();
   }
 
-  public static @MustCallAlias MustCallAliasNormalExit mcaneFactory(InputStream is)
+  public static @MustCallAlias MustCallAliasNormalExit mcaneFactory(@MustCallAlias InputStream is)
```

Comment:
@kelloggm I thought we report a warning somewhere when you write `@MustCallAlias` just on a parameter but not the return, or vice versa?  If not, we should.

Reply:
I don't think that we have any special handling for this case. Because `@MustCallAlias` is a poly annotation, we do get warnings sometimes (if it is placed in a way that is inconsistent with the general poly rules), but (as is clear from this example) not always.

I agree that it would be good for the RLC to warn if there is an `@MustCallAlias` annotation on a parameter but not a return and vice-versa, because that's the only reasonable way to use the annotation.

Reply:
Makes sense, opened #6376 to track